dashboard/src/router.js

```diff
@@ -14,6 +14,11 @@ const routes = [
         name: 'MyProfile',
         component: () => import('@/pages/MyProfile.vue'),
       },
+      {
+        path: '/my-talks',
+        name: 'My Talks',
```

Comment:
let's go with `my-proposals` instead of `my-talks`? nitpicky comment but the difference is a bit jarring (talks pointing to proposals page)

Reply:
done